pkg/kopia/utils.go

```diff
@@ -184,3 +202,95 @@ func GetDataStoreGeneralMetadataCacheSize(opt map[string]int) int {
 	}
 	return defaultDataStoreGeneralMetadataCacheSizeMB
 }
+
+// SetLabelsToPodOptionsIfRequired sets labels to PodOptions
```

Comment:
can you please also add in which case updateNeeded becomes `true`?

Reply:
Will Add !

Reply:
Those functions were not needed in Kanister, so removed them.